### ISSUE TITLE
Telemetry support

### ISSUE THREAD

**Issue by armor66** [ExpressLRS/Handset#22]
Hi, can you please make it clear if it possible to get the rest telemetry data from FC. Why "TLMheader" is always 0x01? I guess that would be 0x14 for link statistics and 0x02, 0x08, 0x21 for gps, battery and flight mode respectively. Is it ELRS version limitation?

**Comment by JBKingdon** [comment]
Hi, I haven't used the extended telemetry from the FC so never implemented support in the handset code. There's quite a lot of complexity you'd need to add to do that, but it should be possible to port it across from mainline elrs if you are motivated.

**Comment by armor66** [comment]
Ok. I just saw the comment: "if (TLMheader == 1) // they changed the header value for new telem support" (It confused me, who are "they"). Does this have to do with SX1280 files? C++ isn't my thing, but anyway I'll try.

**Comment by JBKingdon** [comment]
That's for dealing with the "basic" telemetry that comes from the receiver itself, things like RSSI, SNR and LQ. "They" in this case means the rest of the devs on the main elrs project. They changed the constant for the packets so I had to update the handset code to match for compatibility mode. For the most part I don't run mainline elrs, I have my own fork to experiment with. I added compatibility mode for a friend so that he could use standard elrs on his receivers (since my fork also requires a custom version of betaflight...)
